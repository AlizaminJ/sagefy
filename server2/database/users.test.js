const { getUserByName } = require('./users')

describe('db: users', () => {
  describe('#getUserById', () => {
    test('should getUserById', () => {
      expect(true).toBe(false)
    })
  })

  describe('#getUserByEmail', () => {
    test('should getUserByEmail', () => {
      expect(true).toBe(false)
    })
  })

  describe('#getUserByName', () => {
    test('should getUserByName', async () => {
      const user = await getUserByName('doris')
      expect(user).toMatchSnapshot()
      expect(user.name).toBe('doris')
    })

    test('should not getUserByName', async () => {
      const user = await getUserByName('fantasia')
      expect(user).toBe(null)
    })
  })

  describe('#getUser', () => {
    test('should getUser', () => {
      expect(true).toBe(false)
    })
  })

  describe('#listUsers', () => {
    test('should listUsers', () => {
      expect(true).toBe(false)
    })
  })

  describe('#listUsersByUserIds', () => {
    test('should listUsersByUserIds', () => {
      expect(true).toBe(false)
    })
  })

  describe('#insertUser', () => {
    test('should insertUser', () => {
      expect(true).toBe(false)
    })
  })

  describe('#updateUser', () => {
    test('should updateUser', () => {
      expect(true).toBe(false)
    })
  })

  describe('#updateUserPassword', () => {
    test('should updateUserPassword', () => {
      expect(true).toBe(false)
    })
  })

  describe('#anonymizeUser', () => {
    test('should anonymizeUser', () => {
      expect(true).toBe(false)
    })
  })
})
