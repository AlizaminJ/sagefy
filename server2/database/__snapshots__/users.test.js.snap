// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`db: users #getUserByName should getUserByName 1`] = `undefined`;
